documentation/src/main/markdown/packagemgrs/docker/releasenotes.md

```diff
@@ -1,5 +1,11 @@
 # [docker_inspector_name] Release notes
 
+## Version 11.1.0
+
+### Dependency update
+
+* Changed [docker_inspector_name]'s JAR signing authority from Synopsys, Inc to Black Duck Software, Inc.
```

Comment:
Should the first Inc also have a period?

Reply:
I think it should, good point. Thanks.

Reply:
@niravrsynopsys this should be moved to Changed features not dependency updates cc: @cpottsbd

Reply:
I wasn't sure this qualifies as a feature change. My reasoning was a certificate is a build time dependency.

Reply:
Essentially if they have to update the certificate acceptance (some customers do that), it is in fact a change that they need to make

Reply:
Read this - https://documentation.blackduck.com/bundle/detect/page/downloadingandinstalling/verification.html

Reply:
Ok I see, I'll move it to "Changed Features"